theia-extensions/theia-blueprint-product/package.json

```diff
@@ -18,7 +18,7 @@
   },
   "theiaExtensions": [
     {
-      "frontendElectron": "lib/browser/theia-blueprint-frontend-module",
+      "frontend": "lib/browser/theia-blueprint-frontend-module",
       "electronMain": "lib/electron-main/theia-blueprint-main-module",
       "frontend": "lib/browser/theia-blueprint-frontend-module",
```

Comment:
This now contains the same frontend entry twice. Is this correct?

Reply:
I just checked, there was a mistake in merging the remote branch, after that the frontend entry was listed twice, but I resolved that with commit [`Update Theia versions to 1.38.0`](https://github.com/eclipse-cdt-cloud/cdt-cloud-blueprint/pull/59/commits/42bfa9583be7119246a55af585e4a0e1ede8a235).